### DISCUSSION TITLE
Provide the `ClassName` of the parent module

### DISCUSSION THREAD

**Discussion by Pszz** [family/connectkit#321] [Ideas]
Can some public classNames be opened? The current level of customization is not high enough.

I can't modify Modal's Close button.

For example, our design style: Windows98.

**Comment by lochie** [comment]
We currently have the ID `#__CONNECTKIT__`  on the modal wrapper, but we could definitely use more ways to target individual elements across the modal for more hi-fi customisation. This isn't currently on our roadmap but you're welcome to create a PR with an update to include unique IDs on elements across the modal (these would require to be prefixed to avoid styling contamination).

Also it's worth noting for your use case that we have an in-built `web95` theme you could utilise which has similar vibes to Windows98.
```tsx
<ConnectKitProvider theme="web95">
  /* Your App */
</ConnectKitProvider/>
```
<img width="726" alt="CleanShot —11-22 at 10 40 50@2x" src="https://github.com/family/connectkit/assets/1930210/b3301e08-06ec-45f9-93b3-8a772af9dc1c">

**Reply by Pszz** [reply]
@lochie Thank you for your answer.

This is only similar, but it cannot achieve a 1:1 restoration of the design drawing.

Only exists at the root node with id `#__CONNECTKIT__ ` It's not enough,

When I need to modify the close button at the top, I need to
```css
#__CONNECTKIT_> Div>div>div>div>div>div: first child button{
    background-color: red;
}
```
But I can't be sure if its hierarchy will change. Once it changes, it is unimaginable.

I think it's possible to disclose ClassNames of several parent structures


```css
#__CONNECTKIT_ .header {}
#__CONNECTKIT_ .content {}
#__CONNECTKIT_ .footer {}
```

So all I need is

```css
#__CONNECTKIT_ .header>button{
    background-color: red;
}
```

**Comment by Pszz** [comment]
Of course, you can also componentize the content and divide it into small parts, so users can customize parts.

This part is well done, but it's harder to customize the interior of the pop-ups
`https://docs.family.co/connectkit/connect-button#connect-button-example`
